### ISSUE TITLE
Use of PTv3

### ISSUE THREAD

**Issue by FailFailMakeUp** [Pointcept/PointTransformerV3#7]
Thanks for your amazing work!
I'm following the Pointcept repo to apply PTv3, I use the ScanNet200 pre-trained weight from huggingfance, and want to transfer to my task.
In my task, I have only the indoor scene pointcloud of shape [N, 3], representing the XYZ of each point, then how should I fit the input of PTv3?

**Comment by Gofinge** [comment]
In this case, you might need to retrain a model for fine-tuning.

**Comment by hsekki** [comment]
have you implemented the fine tuning in your custom dataset? i also have [N,3] data. Thank you

**Comment by FailFailMakeUp** [comment]
It seems that PTv3 applies <vertax normal> which is inaccessible with point-only data (without face information).
Maybe another method (I turn to SoftGroup for Instance Segmentation) instead of PTv3 that doesn't apply vertax normal is suitable.
For the missing of color, I simply fill with RGB[128, 128, 128] or normalized, [0, 0, 0].
It is worth noting that without the color information the segmatation performance drops a lot= =

**Comment by Gofinge** [comment]
Hi, the input in Pointcept is fully controlled by config, just edit "feat_key" in "Collect" function. It is not recommended to directly apply a model trained with full features while testing with part of them. 

Of course, custom datasets support all features in Pointcept (or some small modification might needed), further discussion can refer here (https://github.com/Pointcept/Pointcept/issues/108)
